### PR TITLE
Add hglabor

### PR DESCRIPTION
**Pull request by mooziii**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated metadata.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [x] No changes were made to the file's formatting (4 spaces per tab).
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [x] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/metadata.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string, which should match the folder name *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [x] `primaryAddress`: an address *(ex. `my.server`)*
  - [x] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]` - Must be versions supported by Lunar Client)*
  - [x] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18` - Must be versions supported by Lunar Client)*
  - [x] `primaryColor`: a hexademical color code that primarily distiguishes the server *(ex. `#00FFFF`)* 
  - [x] `secondaryColor`: a hexademical color code that accompanies the `primaryColor` of the server *(ex. `#FF0000`)*
  - [x] `primaryRegion`: the primary region where your server operates in *(ex. `NA`)*
  - [x] `gameTypes`: a list of games that describe the content on your server, must be a max of 3 listed *(ez. `["PVP", "UHC", "HCF"]`)*

#### Logo
* [x] My logo is a `png` file.
* [x] I have uploaded my logo to my server folder *(ex. `lunarnetwork`)* and named it `logo.png`.
* [x] My logo has a transparent background and is square (1:1 aspect ratio).
* [x] My logo is `512` pixels in width and height.
* [x] My logo is my own property and complies with relevant copyright/privacy laws.

#### Background
* [x] My background is a `png` file.
* [x] I have uploaded my background to my server folder *(ex. `lunarnetwork`)* and named it `background.png`.
* [x] My background has an aspect ratio of 16:9.
* [x] My background is atleast `1920` pixels in width and `1080` pixels in height (can be bigger, but must remain the same aspect ratio).
* [x] My background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.

### CODE REVIEW

**Review by itsJordan — CHANGES_REQUESTED** [review]
~~Please adjust your background to meet the size requirements (1920x1080 or above in 16:9 ratio)~~
^ Ignore that, my view was incorrect.

File: servers/hglabor/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "secondaryColor": "#ffffff",
+    "primaryLanguage": "de",
+    "languages": [
+        "en",
+        "de"
+    ],
+    "primaryRegion": "EU",
+    "gameTypes": [
+        "PvP",
+        "Training",
```
Comment (itsJordan) [inline servers/hglabor/metadata.json:25]:
```suggestion
```

Comment (itsJordan) [reply servers/hglabor/metadata.json:25]: Training is not one of the allowed types.

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please rename your 'icon.png' to 'logo.png' and upload a background that is more relevant to your server.

**Comment by mooziii** [comment]
changed it

**Review by itsJordan — APPROVED** [review]

**Review by itsJordan — CHANGES_REQUESTED** [review]
please remove your server logo from your background.

**Comment by mooziii** [comment]
Nah I dont have a good Background atm. May reopen once I have one

**Pull request closed without merging**